src/generic/Router.ts

```diff
@@ -83,6 +84,17 @@ export class GenericWebhooksRouter {
         const router = Router();
         router.all(
             '/:hookId',
+            helmet({
+                contentSecurityPolicy: {
+                    useDefaults: true,
+                    directives: {
+                        default: 'self',
+                        sandbox: ''
+                    }
+                },
+                frameguard: { action: 'deny'},
```

Comment:
```suggestion
                xFrameOptions: { action: 'deny'},
```

Does the same thing but hints better at what header it sets.